bindings/wasm/examples/browser/private_tangle.js

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import {Client, Config, Document, KeyType, Network} from "../../web/identity_wasm.js";
+import {
+    logObjectToScreen,
+    logToScreen,
+} from "./utils.js";
+
+/**
+    This example shows how a DID document can be created on a private tangle.
+    It can be run together with a local hornet node.
+    Refer to https://github.com/iotaledger/one-click-tangle/tree/chrysalis/hornet-private-net
+    for setup instructions.
+**/
+export async function createIdentityPrivateTangle(log = true) {
+    if (log) logToScreen("Identity creation started...");
+    if (log)
+        logToScreen("This might take a few seconds to complete proof of work!");
+
+    // This is the `networkID` defined in hornet's node configuration.
+    const network = Network.from_name("private-tangle");
+
+    // Create a DID Document (an identity).
+    const { doc, key } = new Document(KeyType.Ed25519, network.toString());
+
+    // Sign the DID Document with the generated key.
+    doc.sign(key);
+
+    // Create a client configuration and set the custom network.
+    const config = new Config();
+    config.setNetwork(network);
+
+    // This URL points to the REST API of the locally running hornet node.
+    config.setNode("http://127.0.0.1:14265/");
+
```

Comment:
permanode should be set too?

Reply:
But for the example it's not required and I don't think there's a one-click permanode available, right?

Reply:
afaik resolving a DID requires a permanode. 
You can run your own permanode separately [chronicle](https://github.com/iotaledger/chronicle.rs). And there's an issue for adding it to one-click-tangle [here](https://github.com/iotaledger/one-click-tangle/issues/46). We still didn't test that yet. Eike @eike-hass had some progress regarding that.

Reply:
Since the example is just publishing sure but it's probably necessary if you're going to use a private tangle. Maybe it's worth at least mentioning it in a comment that it is required for resolution when using a custom network and "this is how you would set it".

Reply:
Permanodes are not needed, but they guarantee that an existing identity is found.